README.md

```diff
@@ -25,9 +25,9 @@ The ```main``` branch of this repository contains samples configured for the lat
 
 ## Requirements
 
-* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) 100.12.0 (or newer)
-* [ArcGIS Runtime Toolkit for iOS](https://github.com/Esri/arcgis-runtime-toolkit-ios) 100.12.0 (or newer)
-* Xcode 12.0 (or newer)
+* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) 100.13.0 (or newer)
+* [ArcGIS Runtime Toolkit for iOS](https://github.com/Esri/arcgis-runtime-toolkit-ios) 100.13.0 (or newer)
+* Xcode 13.0 (or newer)
```

Comment:
I'm curious why this was increased? Xcode 13.0 isn't needed to use the 100.13 release. Is the sample app using Swift language features that requires Xcode 13?

Reply:
I don't have a good reason. We don't use new features in Swift 5.5 or Xcode 13, and I suggested Vivian to bump the version, thinking that we can revert the change if needed in PR review, rather than not making the change and let it slip.

A minor reason to use Xcode 13 is that it's more stable for SwiftUI preview, but that doesn't make it a requirement.